.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor Config
```

Comment:
Do we really need this?

Reply:
I can't see a reason why anyone wouldn't use EditorConfig?